core/src/main/java/org/elasticsearch/search/fetch/subphase/highlight/PostingsHighlighter.java

```diff
@@ -44,13 +46,15 @@
 import java.util.Locale;
 import java.util.Map;
 
+@Deprecated
 public class PostingsHighlighter implements Highlighter {
+    private static DeprecationLogger deprecationLogger = new DeprecationLogger(Loggers.getLogger(PostingsHighlighter.class));
 
     private static final String CACHE_KEY = "highlight-postings";
 
     @Override
     public HighlightField highlight(HighlighterContext highlighterContext) {
-
+        deprecationLogger.deprecated("[postings] highlighter is deprecated, please use [unified] instead");
```

Comment:
Thanks, I tried to limit to one per request in this change:
https://github.com/elastic/elasticsearch/pull/25073/commits/31ee14516ae2fa1baa14089b62ed8018120e58e5
Can you take another look ?